src/js/components/Video/doc.js

```diff
@@ -31,3 +31,31 @@ export const doc = Video => {
 
   return DocumentedVideo;
 };
+
+export const themeDoc = {
+  'global.colors': {
+    description: 'The background color of the video scrubber.',
+    type: 'string',
+    defaultValue: 'light-4',
```

Comment:
I don't think it's quite accurate to say that the default value for `global.colors` is "light-4" or a type string. I think you are trying to say that the color of the Video controls is 'light-4'. Maybe we should add this to the theme as `video.controls.color`?

Reply:
Yes, I agree, I'll refactor the reference of the hardcoded `light-4` to be taken from the theme.